lib/src/bit_array.dart

```diff
@@ -223,6 +223,30 @@ class BitArray implements BitSet {
   Iterable<int> asIntIterable([bool value = true]) {
     return _IntIterable(this, value);
   }
+
+  @override
+  bool operator ==(Object other) {
+    if (identical(this, other)) {
+      return true;
+    }
+    if (other is BitArray &&
+        runtimeType == other.runtimeType &&
+        length == other.length) {
+      for (var i = 0; i < _data.length; i++) {
+        if (_data[i] != other._data[i]) {
+          return false;
+        }
+      }
+      return true;
+    }
+    return false;
+  }
+
+  @override
+  int get hashCode =>
```

Comment:
I'm wondering if both `hashCode` and `==` could be done with one the iterable methods that return either the ints or the 32bit ints. That way they could go inside BitSet. WDYT?

Reply:
I'd just need to change every class from `implements BitSet` to `extends BitSet`. If that's okay, I can change it.

Reply:
Sure, please do so...